stablehlo/transforms/VhloLegalizeToStablehlo.cpp

```diff
@@ -500,6 +500,12 @@ SpecialResult convertSpecial(const OpConversionPattern<VhloOpTy>& pattern,
       if (!stablehloAttr) return specialFailure();
     }
   }
+  if constexpr (std::is_same<VhloOpTy, vhlo::CompositeOpV1>::value) {
+    if (vhloName == "decomposition") {
+      stablehloAttr = convertSymbol(vhloAttr, pattern.getTypeConverter());
```

Comment:
Per my comments on the spec earlier, this is the result of the hackery around function refs being stored as strings. I don't love that we have to do this..

Reply:
To be addressed separately though

Reply:
Ack, thanks for sharing your thought process on this!